pkgs/applications/networking/instant-messengers/ripcord/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchurl, appimageTools }:
+
+let
+  pname = "Ripcord";
+  version = "0.4.18";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://cancel.fm/dl/${pname}-${version}-x86_64.AppImage";
+    sha256 = "19nv5pq8643l57hr5bdgn84rrw3ak2p9s16gr202nvi62flnl44p";
+  };
+
+  appimageContents = appimageTools.extractType2 {
+    inherit name src;
+  };
+in appimageTools.wrapType2 rec {
+  inherit name src;
```

Comment:
Does this not accept `pname`?

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/appimage/default.nix#L56
apparently nope